controllers/provisioners/eks/helpers.go

```diff
@@ -302,6 +302,10 @@ func (ctx *EksInstanceGroupContext) GetAddedTags(asgName string) []*autoscaling.
 		switch strings.ToLower(osFamily) {
 		case OsFamilyWindows:
 			tags = append(tags, ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/node-template/label/kubernetes.io/os", "windows", asgName))
+			tags = append(tags, ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/node-template/label/kubernetes.io/arch", "amd64", asgName))
+			instanceTypeNetworkInfo := awsprovider.GetInstanceTypeNetworkInfo(ctx.GetDiscoveredState().GetInstanceTypeInfo(), configuration.InstanceType)
+			numberOfIps := *instanceTypeNetworkInfo.Ipv4AddressesPerInterface - 1
```

Comment:
Should we use this instead:
```golang
numberOfIps := aws.Int64Value(instanceTypeNetworkInfo.Ipv4AddressesPerInterface) - 1
```

Reply:
It's also possible for `GetInstanceTypeNetworkInfo` to return nil, so we might want a nil check to avoid a nil pointer? Not sure if feasible for this to happen